### PR TITLE
[DOC-343] [DOC-286] [DOC-354] [DOC-313] DDL atomicity flags, Catalog flag, bitmaps scan, Truncate table support in PITR

### PR DESCRIPTION
**Pull request by aishwarya24**
Preview for DDL atomicity: https://deploy-preview-22302--infallible-bardeen-164bc9.netlify.app/preview/reference/configuration/yb-tserver/#ddl-atomicity-flags

### CODE REVIEW

File: docs/content/preview/manage/backup-restore/point-in-time-recovery.md [hunk 1]
```diff
@@ -262,6 +262,5 @@ YugabyteDB Anywhere [supports PITR](../../../yugabyte-platform/back-up-restore-u
 
 ### Other limitations
 
-- The `TRUNCATE` command is disallowed for databases with a snapshot schedule. Tracking issue: [7129](https://github.com/yugabyte/yugabyte-db/issues/7129).
```
Comment (fizaaluthra) [inline docs/content/preview/manage/backup-restore/point-in-time-recovery.md:265]: Reviewed this file, LGTM

**Comment by myang2021** [comment]
The DDL atomicity flags LGTM!

**Review by kai-franz — APPROVED** [review]
TOAST flag LGTM!

**Comment by aishwarya24** [comment]
Created https://github.com/yugabyte/yugabyte-db/pull/22566

**Pull request closed without merging**